traits_enaml/testing/enaml_test_assistant.py

```diff
@@ -37,12 +40,12 @@ def print_enaml_widget_tree(widget, level=0):
     """
     level = level + 4
     if level == 0:
-        print
-    print ' '*level, widget
+        print()
+    print(six.text_type(' '*level), six.text_type(widget))
```

Comment:
Is there any reason why this can't simply be: `print(' '*level, widget)`?

Reply:
I was seeing errors about expecting unicode but getting string inputs, which might actually be related to how we are testing these helper functions and not a problem with the functions themselves. I'll try opening the StreamIO with bytes and see if the problem goes away.